client/token/decode.go

```diff
@@ -0,0 +1,48 @@
+package token
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+)
+
+var errInvalidToken = errors.New("invalid project token")
```

Comment:
```suggestion
var ErrInvalidToken = errors.New("invalid project token")
```

I think we should export this error now that it is a public package.